root/multicore/testSetAddressLoop.bat

```diff
@@ -0,0 +1,12 @@
+@echo off
+
+set COUNTER=0
+:loop
+echo "Run number %COUNTER%"
+testSetAddress.exe
+if errorlevel 1 (
+  echo "ERROR: The executable did not complete successfully!";
+  exit /b 1;
+)
+set /a COUNTER+=1
+if %COUNTER% LSS 33 goto loop
```

Comment:
Are batch files still a thing? We have powershell for years!

Reply:
Right, but there is only one batch script...

Reply:
And since cling (llvm/clang) still has known open bugs and suffers from incompatibilities wit MS runtime, we need to disable tests until ROOT/cling gets fixed